### ISSUE TITLE
ownCloud Client 1.8.1 Release

### ISSUE THREAD

**Issue by dragotin** [owncloud/client#3198]
beta1, RC1 and RC2 were already done before starting this checklist.

Release-1 Week:
* [x] Ensure up-to-date dependencies (e.g. [latest Qt version](http://qt-project.org/downloads#qt-lib) is installed on the machine and picked up (cmake output) (Windows still on 4.8.0 + patches, to be updated after 1.8.x).
* [x] Check nightly builds are up and running, that is Jenkins jobs ownCloud-client-linux, ownCloud-client-osx and ownCloud-client-win32 all green.
* [x] Build branded clients through the scripting machine and smoke test one or two branded clients (especially with predefined url)
* [x] Upload a nightly build of the windows version to virustotal.com
  * Contact AV vendors whom's engine reports a virus
* [x] Documentation should be online before the release http://doc.owncloud.org/desktop/1.X/
* [x] QA goes over https://github.com/owncloud/mirall/wiki/Testing-Scenarios
* [x] Communicate the release schedule on mailinglist release-coordination@owncloud.com
* [x] Give a high level overview of the upcoming new features, changes etc.
* [x] Make sure to have `client/ChangeLog` updated
 * use `git log --format=oneline v<lastrelease>...master` if your memory fails you
* [x] Ensure marketing is aware and prepared for the release (social, .com website, cust. communications)
* [x] Inform GCX knows the next version is about 1 week out (gcx@owncloud.com)

Day before Release:
* [x] Check the translations coming from transifex: All synchronized?
* [x] Run the tx.pl scripts on the final code tag
* [x] Run ```make test```
* [x] Run smashbox on the final code tag
* [x] Inform product management and marketing that we are 1 day out

On Release Day:
* [x] Add last updates to Changelog in the client source repository.
* [x] Pick SHAs to build from (client) (automated by `ownCloud-client-source` jenkins job)
 * `git rev-parse master` returns SHA of current checkout
* [x] Double check ```VERSION.cmake```: Check the version number settings and suffix (beta etc.) to be removed.
* [x] Create tar ball (automated by `ownCloud-client-source` jenkins job)
* [x] Copy the source tarball from the daily to the stable dir on download.o.o 
* [x] Announce the source tarball on the packaging mailing list packaging@owncloud.org
* [x] Build Windows packages
* [x] Build Mac OS X packages
 * Sign package `$srcdir/admin/osx/sign_dmg.sh file.dmg <identity>`, result is in signed_file.dmg (automated now)
* [x] Build Linux packages by running the jenkins job ownCloud-client-linux with proper parameters
 * Update [OBS repository](https://build.opensuse.org/project/show?project=isv%3AownCloud%3Adesktop) `isv:ownCloud:desktop` (or `isv:ownCloud:community:testing` for RC/Beta)
 * Check if patches still apply in the linux packages
* [x] Trigger rebuild of the branding packages, inform juergen that a new tarball is available.
* [x] Check if the following packages are on download.owncloud.com/desktop/stable:
  * Windows binary package
  * Mac binary package
  * source tarballs
* [x] Re-download Win and Mac builds and check signature.
* [x] Perform smoke test (Install, make sure it does not explode, and check if all version indicators are correct)
  * If all is good, tag build SHAs from step 2
* Create an submitrequest to the www repository to update the install page and the changelog on owncloud.org
* [x] Update ASCII changelog on http://download.owncloud.com/download/changelog-client 
* [x] Keep the packaging mailinglist packaging@owncloud.org informed and announce the final sources.
* [x] Create git tags in github client repository.
* [x] Update the updater script ```clientupdater.php```
* [x] Send out Social (tweet, blog, other)
* [x] Send out customer communication (no communication outside newsletters for minor releases)
* [x] Inform GCX that the new version is released (gcx@owncloud.com)
* [x] Take pride and celebrate!

**Comment by danimo** [comment]
Updater is armed.
